documentation/examples/remote_storage/remote_storage_adapter/main.go

```diff
@@ -141,6 +146,9 @@ func parseFlags() *config {
 	)
 	flag.StringVar(&cfg.listenAddr, "web.listen-address", ":9201", "Address to listen on for web endpoints.")
 	flag.StringVar(&cfg.telemetryPath, "web.telemetry-path", "/metrics", "Address to listen on for web endpoints.")
+	flag.StringVar(&cfg.logLevel, "log.level", "",
+		"The log level to be set for promlog",
```

Comment:
@sipian can you add the log levels to this comment, that way when a user uses option '-h' they will know what is available options for this flag?
error, warn, info and debug